pkg/tests/tasks/traffic/egress/egress_gateways_tls_file_mount_test.go

```diff
@@ -32,55 +35,61 @@ func TestTLSOrigination(t *testing.T) {
 		t.Log("  1) Egress gateway TLS Origination")
 		t.Log("  2) MTLS Origination with file mount (certificates mounted in egress gateway pod)")
 
-		ns := "bookinfo"
 		t.Cleanup(func() {
-			app.Uninstall(t, app.Sleep(ns))
+			app.Uninstall(t, app.Sleep(ns.Bookinfo))
 		})
 
 		ossm.DeployControlPlane(t)
 
 		t.LogStep("Install sleep pod")
-		app.InstallAndWaitReady(t, app.Sleep(ns))
+		app.InstallAndWaitReady(t, app.Sleep(ns.Bookinfo))
 
 		t.NewSubTest("Egress Gateway without file mount").Run(func(t TestHelper) {
 			t.Log("Perform TLS origination with an egress gateway")
 
-			t.LogStep("Create ServiceEntry for istio.io, port 80 and 443")
-			oc.ApplyString(t, ns, ExServiceEntry)
-			t.Cleanup(func() {
-				oc.DeleteFromString(t, ns, ExServiceEntry)
+			t.LogStep("Install external nginx")
+			app.InstallAndWaitReady(t, app.NginxExternalTLS(ns.MeshExternal))
+
+			t.LogStep("Make sure that mesh external namespace is not discovered by Istio - it would happen if mesh-external namespaces was added to the SMMR")
```

Comment:
I think the last comment is better to be added as a comment `it would happen if mesh-external namespaces was added to the SMMR` is not need as step name or explanation for the step

Reply:
Done